x-pack/plugins/aiops/README.md

```diff
@@ -0,0 +1,9 @@
+# aiops
+
+The Kibana AIOps plugin provides APIs and components for the "Explain log rate spikes" UI.
```

Comment:
Is it worth making this description a bit more future proof, so we don't need to change it if we add e.g. code currently used in APM correlations, or components for searching to explain metric spikes?

Reply:
Updated the text in [e523670](https://github.com/elastic/kibana/pull/131317/commits/e523670f04f802b216cff74d7db9e9bc6bcc4174).